ui/raidboss/data/03-hw/raid/a3s.js

```diff
@@ -144,12 +152,14 @@
         if (marker1 == marker2) {
           return {
             en: 'Repel: close to ' + data.ShortName(partner),
+            fr: 'Repoussement: Rapprochez-vous de ' + data.ShortName(partner),
             cn: '同极：靠近' + data.ShortName(partner),
           };
         }
 
         return {
           en: 'Attract: away from ' + data.ShortName(partner),
+          fr: 'Attirement: Eloignez-vous de ' + data.ShortName(partner),
```

Comment:
Attraction. _Attirement_ does not exist.

Reply:
je corrige